lib/features/mailbox/presentation/mixin/mailbox_widget_mixin.dart

```diff
@@ -60,11 +60,11 @@ mixin MailboxWidgetMixin {
         MailboxActions.markAsRead,
       MailboxActions.move,
       MailboxActions.rename,
-      MailboxActions.delete,
       if (mailbox.isSubscribedMailbox)
         MailboxActions.disableMailbox
       else
-        MailboxActions.enableMailbox
+        MailboxActions.enableMailbox,
+      MailboxActions.delete
```

Comment:
```suggestion
        MailboxActions.enableMailbox,
        MailboxActions.delete
```

Reply:
The `enableMailbox` action is in an `else` condition so it needs to be indented. The `delete` action does not need to be indented.